calculator.c

```diff
@@ -1129,7 +1123,12 @@ void handleSorts(BranchPath *curNode) {
 ABSL_MUST_USE_RESULT BranchPath *initializeRoot() {
 	BranchPath *root = malloc(sizeof(BranchPath));
 
-	checkMallocFailed(root);
```

Comment:
I think there was a mistake in your merge, as `checkMallocFailed` should probably be kept here.

Reply:
Good catch. Done